GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -510,39 +472,21 @@ extension CameraViewController {
         })
     }
     
+    /**
+     Replaces captured images stack content with new images.
+     
+     - parameter images: New images shown in the stack. (Last image will be shown on top)
```

Comment:
Replaces **the** captured images stack content with new images.

 - parameter images: New images **to be** shown in the stack. (Last image will be shown on top)